includes/templates/reader-activation-emails/otp.php

```diff
@@ -9,49 +9,54 @@
+	<div class="wp-block-group has-secondary-background-color has-' . esc_attr( $secondary_text_color ) . '-color has-text-color has-background has-link-color" style="background-color:' . esc_attr( $secondary_color ) . ';padding-top:56px;padding-right:56px;padding-bottom:56px;padding-left:56px">
 	<!-- wp:site-logo {"width":50} /-->
 	<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
 	<h2 class="wp-block-heading" style="font-style:normal;font-weight:400">' . __( 'Sign in', 'newspack' ) . '</h2>
 	<!-- /wp:heading -->
-
 	<!-- wp:paragraph -->
 	<p>' . __( 'Use the following code to login to your account:', 'newspack' ) . '</p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:paragraph {"fontSize":"x-large"} -->
 	<p class="has-x-large-font-size"><code>*MAGIC_LINK_OTP*</code></p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
 	<p style="font-style:normal;font-weight:400">' . __( 'You can also log into your account by clicking the following button:', 'newspack' ) . '</p>
 	<!-- /wp:paragraph -->
-
 	<!-- wp:buttons {"layout":{"type":"flex","orientation":"horizontal","flexWrap":"nowrap","justifyContent":"left"},"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}},"fontSize":"medium"} -->
-	<div class="wp-block-buttons has-custom-font-size has-medium-font-size" style="font-style:normal;font-weight:400"><!-- wp:button {"textAlign":"center","style":{"color":{"background":"' . esc_attr( $primary_color ) . '"},"border":{"radius":"4px"}}} -->
-	<div class="wp-block-button"><a class="wp-block-button__link has-background has-text-align-center wp-element-button" href="*MAGIC_LINK_URL*" style="border-radius:4px;background-color:' . esc_attr( $primary_color ) . '">' . sprintf( /* Translators: s: site title. */ __( 'Continue to %s', 'newspack' ), '*SITE_TITLE*' ) . '</a></div>
+	<div class="wp-block-buttons has-custom-font-size has-medium-font-size" style="font-style:normal;font-weight:400"><!-- wp:button {"textAlign":"center","textColor":"' . esc_attr( $primary_text_color ) . '","style":{"color":{"background":"' . esc_attr( $primary_color ) . '"},"border":{"radius":"4px"},"elements":{"link":{"color":{"text":"' . esc_attr( $primary_text_color ) . '"}}}}} -->
+	<div class="wp-block-button"><a class="wp-block-button__link has-' . esc_attr( $primary_text_color ) . '-color has-text-color has-background has-link-color has-text-align-center wp-element-button" href="*MAGIC_LINK_URL*" style="border-radius:4px;background-color:' . esc_attr( $primary_color ) . '">Continue to *SITE_TITLE*</a></div>
```

Comment:
Is the removal of translation function (`__()`) intentional here?

Reply:
This was not intentional. Fixed in https://github.com/Automattic/newspack-plugin/pull/3033/commits/6510cc350130095d99547fc49098875bdd880ede